src/trie.c

```diff
@@ -54,6 +55,7 @@ int add_node(char current, trie_t *t)
 
     if (t->children[c] == NULL)
         t->children[c] = new_trie(current);
+    
```

Comment:
Looks like someone's editor doesn't strip trailing whitespace. Please revert this change so that this PR doesn't touch this file.

Reply:
Actually I figured out how to do this with git checkout but the issue is I think Ben actually updated the src file in one of the commits (to reflect the new magic numbers thing brought up in a different PR)  so we can't have a PR that doesn't modify the src/trie.c file. It either has to be like the way it is now (with the magic numbers not existing) or the way it is above (with the white space) I'm sorry but I really can't fix this (I've gone through all of our commits to see if there's a commit that I could revert to that would work and legitimately none of them work). So if you want to keep it like it is now I can do that or if you want me to change it back to the edit with the white space and without the magic numbers I can do that to, just let me know.

Reply:
@mpcsh

Reply:
Can you just manually make a new commit that fixes the whitespace?